src/Network/Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -190,6 +190,13 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
         [Alias("UserAssignedIdentity")]
         public string UserAssignedIdentityId { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Application Gateway Identity to be assigned to Application Gateway.")]
+        [ValidateNotNullOrEmpty]
+        public PSManagedServiceIdentity Identity { get; set; }
```

Comment:
@akshaysngupta it looks like there exists a `-UserAssignedIdentityId` parameter in this cmdlet that does exactly what `New-AzApplicationGatewayIdentity` does, and they aren't in separate parameter sets, so users will be able to provide a value for both. Does it make sense to have this parameter if it does the same thing as the new cmdlet you're adding, but doesn't require an additional cmdlet call?